tests/test_custom_objects_rest.py

```diff
@@ -0,0 +1,100 @@
+"""
+Test that all fields can be replicated for a stream that is a custom object (REST API)
+"""
+from datetime import datetime, timedelta
+
+from tap_tester import connections, menagerie, runner
+
+from base import SalesforceBaseTest
+
+
+class SalesforceCustomObjects(SalesforceBaseTest):
+    """Test that all fields can be replicated for a stream that is a custom object"""
+
+    # Note: custom record as seen Aug 9, 2023
+    # TODO do we need to hard code start date to 08-04?
+    # 'CreatedDate': '2023-08-04T20:13:50.000000Z'
+    # 'SystemModstamp': '2023-08-04T20:13:50.000000Z',
+    # 'LastModifiedDate': '2023-08-04T20:13:50.000000Z',
+    # 'LastReferencedDate': '2023-08-09T20:19:45.000000Z',
+    # 'LastViewedDate': '2023-08-09T20:19:45.000000Z',
+    start_date =  (datetime.now() + timedelta(days=-1)).strftime("%Y-%m-%dT00:00:00Z")
+
+    @staticmethod
+    def expected_sync_streams():
+        return {
+            'TapTester__c',  # 1 created record present, spike on CRUD?
+            # 'TapTester__Share',  # TODO no data, fix or omit?
+        }
+
+    def custom_objects_test(self):
+        """
+        Verify that all fields can be replicated for a custom  stream
+
+        PREREQUISITE
+        Create a custom object and at least one data record for that cusotm object
+        """
+
+        expected_streams = self.expected_sync_streams()
+
+        # instantiate connection
+        conn_id = connections.ensure_connection(self, original_properties=False)
```

Comment:
In the new framework we generally aren't using original properties any longer as this is to change start date which we made a property we can easily override instead.

Reply:
Will remove along with update to new base file.